dsmr_parser/parsers.py

```diff
@@ -93,9 +93,14 @@ def parse(self, telegram_data, encryption_key="", authentication_key=""):  # noq
             for match in matches:
                 try:
                     dsmr_object = parser.parse(match)
-                except Exception:
+                except ParseError:
                     logger.error("ignore line with signature {}, because parsing failed.".format(signature),
                                  exc_info=True)
+                    if throw_ex:
+                        raise
+                except Exception as err:
+                    logger.error("Unexpected {}: {}".format(type(err), err))
```

Comment:
No need to change, but in general when using the logging module i pass the variables as arguments:
```
logger.error("Unexpected %s: %s", err, err)
```

Here is a nice blogpost about it https://www.palkeo.com/en/blog/python-logging.html#use-a-format-string-with-arguments

In short: when doing things with the logs (for example Sentry), it allows for better grouping etc